### PR TITLE
UI blackjack

### PR DESCRIPTION
**Pull request by BrassoiCatalin**
Made it so you can bet an amount in Blackjack game.

### CODE REVIEW

**Review by TinelValentin — APPROVED** [review]

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs [hunk 1]
```diff
@@ -1,11 +1,67 @@
...
+            TotalWin = 0;
+            InitializeCommands();
+        }
+        #endregion
 
+        #region Private Fields...
+        private int _totalBet;
+        private int _ownedMoney;
+        private int _totalWin;
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs:37]: Unnecessary new line

**Review by Sorin37 — CHANGES_REQUESTED** [review]

**Review by TinelValentin — APPROVED** [review]

File: SevenSlots/SevenSlots/SevenSlots/Model/Player.cs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        {
+            get => _cardTotal;
+            set
+            {
+                _cardTotal = value;
+                OnPropertyChanged(nameof(CardTotal));
+            }
+        }
+
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/Model/Player.cs]: Unnecessary new line

File: SevenSlots/SevenSlots/SevenSlots/Model/Player.cs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            {
+                _betAmount = value;
+                OnPropertyChanged(nameof(BetAmount));
+            }
+        }
+
+        #endregion
+
+        #region Constructor
+        public Player(string name, int bankroll, int totalbet = 0, int totalwin = 0, int betAmount = 0)
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/Model/Player.cs]: the name of the parameters do not respect the camel case regulations

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs [hunk 4]
```diff
@@ -1,11 +1,183 @@
...
+        #endregion
+
+        #region Private Fields...
+
+        private int _totalBet;
+        private int _ownedMoney;
+        private int _totalWin;
+        List<string> _cards = new List<string>()
+        {
+             "c01", "c02", "c03", "c04", "c05", "c06", "c07", "c08", "c09", "c10", "c11", "c12", "d13", "d01", "d02", "d03", "d04", "d05", "d06", "d07", "d08", "d09", "d10", "d11", "d12", "d13", "h01", "h02", "h03", "h04", "h05", "h06", "h07", "h08", "h09", "h10", "h11", "h12", "h13", "s01", "s02", "s03", "s04", "s05", "s06", "s07", "s08", "s09", "s10", "s11", "s12", "s13"
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs]: I suggest that you would put the strings on separate lines like so:
c01, c02...
d01, d02...
and so on

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs [hunk 5]
```diff
@@ -1,11 +1,183 @@
...
+        private (List<string> dCards, List<string> pCards) DealInitalCards()
+        {
+            List<string> dealerCards = new List<string>();
+            List<string> playerCards = new List<string>();
+
+            Random random = new Random();
+
+            //reshuffleCards();
+
+            for (int i = 0; i < INITIALDEAL; i++)
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs]: why is INITIALDEAL all caps?

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs [hunk 6]
```diff
@@ -1,11 +1,183 @@
...
+            _dealer.HiddenCard = ("../Images/Cards/" + value.dCards[0] + ".bmp");
+            _dealer.Card.Add("../Images/Cards/" + value.dCards[1] + ".bmp");
+
+            _player.Card.Add("../Images/Cards/" + value.pCards[0] + ".bmp");
+            _player.Card.Add("../Images/Cards/" + value.pCards[1] + ".bmp");
+
+            int.TryParse(value.dCards[0].Substring(1).TrimStart('0'), out int dCardValue1);
+            int.TryParse(value.dCards[1].Substring(1).TrimStart('0'), out int dCardValue2);
+
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs]: Unecessary new line

File: SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs [hunk 7]
```diff
@@ -1,11 +1,183 @@
...
+            {
+                cardValue1 = 11;
+            }
+
+            if (cardValue2 == 1 && cardValue1 <= 10)
+            {
+                cardValue2 = 11;
+            }
+
+
```
Comment (Sorin37) [inline SevenSlots/SevenSlots/SevenSlots/ViewModel/BlackjackViewModel.cs]: Unnecessary new line

**Review by Sorin37 — CHANGES_REQUESTED** [review]

**Review by Sorin37 — APPROVED** [review]
Lgtm

**Pull request merged**